packages/client/hmi-client/src/components/widgets/tera-resizable-panel.vue

```diff
@@ -0,0 +1,71 @@
+<!--
+TeraResizablePanel: AVertically resizable panel with a handle along the bottom
+-->
+
+<template>
+	<div class="container" :style="containerStyle">
+		<div class="content">
+			<slot></slot>
+		</div>
+		<div class="resize-handle" @mousedown="startResize"></div>
+	</div>
```

Comment:
We try to:
* use as much HTML5 semantic element to avoid `<div>` soup when looking at the whole app on the browser. And it simplify the CSS as we do not need classname as every components are scoped.
* use CSS properties instead of writing CSS in JavaScript

i.e
```Vue
<section :style="{ '--height': containerHeight + 'px' }">
  <main><slot /></main>
  <div class="resize-handle" @mousedown="startResize"></div>
</section>

<style scoped>
section {
  min-height: 300px; /* Like this you can remove the whole containerStyle() function */
	height: var(--height, 100%);
}

main {
	height: 100%;
}
...
</style>
````